client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -83,14 +86,9 @@ class RatingsAndReviews extends React.Component {
       });
   }
 
-  ratingDetails(averageRating, totalRating) {
-    this.averageRating = averageRating;
-    this.totalRating = totalRating;
-  }
-
   getDefaultReviews() {
-    const { displayedReviews, reviews, product_id } = this;
-    this.props.updateReviews(reviews);
+    const { reviewsData, displayedReviews, reviews, product_id } = this;
+    this.props.updateReviews(reviewsData);
```

Comment:
Please use `updateCache` instead of these methods. So sorry for pointing you down that path 😬

Reply:
Updated! And no worries, cache makes more sense now :)